buildutils/attr_parser.py

```diff
@@ -913,25 +340,21 @@ def main(argv):
         print('Cannot Open Server File!')
         sys.exit(1)
 
+    try:
+        d_file = open(DEF_FILE, 'w', encoding='utf-8')
+    except IOError as err:
+        print(str(err))
+        print('Cannot Open Defines File!')
```

Comment:
would displaying the file name be helpful?

Reply:
yeah that can be changed